### PR TITLE
Create and delete OrgVDC and more

### PR DESCRIPTION
**Pull request by pacogomez**
list of changes are listed in the commits

---
This change is [Reviewable](https://reviewable.io/reviews/vmware/vcd-cli/139)

### CODE REVIEW

File: vcd_cli/disk.py [hunk 1]
```diff
@@ -96,13 +93,19 @@ def list_disks(ctx):
         disks = vdc.get_disks()
         result = []
         for disk in disks:
+            attached_vms = ''
+            if hasattr(disk, 'attached_vms') and \
+               hasattr(disk.attached_vms, 'VmReference'):
+                attached_vms = disk.attached_vms.VmReference.get('name')
```
Comment (sahithi) [inline vcd_cli/disk.py]: There can be multiple VmReference records, right?

File: vcd_cli/profile.py [hunk 2]
```diff
@@ -26,3 +29,31 @@ def profile(ctx):
     """
     profiles = Profiles.load()
     click.echo(yaml.dump(profiles.data, default_flow_style=False))
+
+
+@vcd.command(short_help='current resources in use')
+@click.pass_context
+def pwd(ctx):
```
Comment (sahithi) [inline vcd_cli/profile.py]: To me, profile sounds better than pwd.

File: vcd_cli/vapp.py [hunk 3]
```diff
@@ -524,3 +522,28 @@ def add_disk(ctx, name, vm_name, size, storage_profile):
         stdout(task, ctx)
     except Exception as e:
         stderr(e, ctx)
+
+
+@vapp.command(short_help='set active vApp')
+@click.pass_context
+@click.argument('name', metavar='<name>', required=True)
+def use(ctx, name):
```
Comment (sahithi) [inline vcd_cli/vapp.py]: Now that 'use' command has been added to vApp, I think we should make vAppname parameter optional for all the existing commands in vcd-cli/vapp.py t keep it consistent with org and vdc commands? What do you think?

File: vcd_cli/vdc.py [hunk 4]
```diff
@@ -117,12 +131,153 @@ def use(ctx, name):
...
+    '--allocation-model',
+    type=click.Choice(['AllocationVApp', 'AllocationPool', 'ReservationPool']),
+    required=False,
+    default='AllocationVApp',
+    metavar='<allocation-model>',
+    help='Allocation model.')
+@click.option(
+    '-s',
+    '--storage-profile',
+    'pvdcsp_name',
```
Comment (sahithi) [inline vcd_cli/vdc.py]: I think sp_name is good enough.

File: vcd_cli/vdc.py [hunk 5]
```diff
@@ -117,12 +131,153 @@ def use(ctx, name):
...
+    help='Capacity that is commited to be available.')
+@click.option(
+    '--cpu-limit',
+    required=False,
+    default=0,
+    metavar='<cpu-limit>',
+    type=click.INT,
+    help='Capacity limit relative to the value specified for Allocation.')
+def create(ctx, name, pvdc_name, network_pool_name, allocation_model,
+           pvdcsp_name, pvdcsp_limit, description, cpu_allocated, cpu_limit):
```
Comment (sahithi) [inline vcd_cli/vdc.py]: We should provide support to add multiple storage profiles to an ovdc. Please track a separate Jira task as this change set is already big enough

File: vcd_cli/disk.py (hunk 1 above)
Comment (pacogomez) [reply vcd_cli/disk.py]: I think a VMDK can only be attached to a single VM at any given time, but the underlying API method returns a list of VMs. This is to be consistent with the API. http://pubs.vmware.com/vcd-820/index.jsp#com.vmware.vcloud.api.reference.doc_27_0/doc/types/VmsType.html?resultof=%2522%2561%2574%2574%2561%2563%2568%2565%2564%2576%256d%2573%2522%2520%2522%2561%2574%2574%2561%2563%2568%2565%2564%2576%256d%2522%2520

File: vcd_cli/profile.py (hunk 2 above)
Comment (pacogomez) [reply vcd_cli/profile.py]: we already have `profile` to show current content of the profile file. `profile` will be used in the future as a group command and will have `list`, `info`, `use`, `create`, etc subcommands. But I agree in that we can try to find a better alternative.

File: vcd_cli/vapp.py (hunk 3 above)
Comment (pacogomez) [reply vcd_cli/vapp.py]: Yes, I agree.

File: vcd_cli/vdc.py (hunk 5 above)
Comment (pacogomez) [reply vcd_cli/vdc.py]: I already did, see VCDA-307 and VCDA-308

**Pull request merged**